src/main/java/com/example/demo/entity/Comment.java

```diff
@@ -0,0 +1,37 @@
+package com.example.demo.entity;
+
+import com.example.demo.dto.CommnetDTO;
+import lombok.*;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedBy;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@EntityListeners(AuditingEntityListener.class)
+@Builder
+@Table(name = "COMMENT")
+public class Comment {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long com_id;
```

Comment:
```suggestion
    private Long commentId;
```

자바에서 필드는 카멜 케이스로 써주시면 좋을 듯 해요